blockchain/reactor.go

```diff
@@ -222,6 +273,7 @@ FOR_LOOP:
 			peer := bcR.Switch.Peers().Get(peerID)
 			if peer != nil {
 				bcR.Switch.StopPeerForError(peer, errors.New("BlockchainReactor Timeout"))
+				bcR.MarkPeer(peer, false, 1, p2p.Bad)
```

Comment:
how does marking bad play with a stopped peer ?

Reply:
Thanks, I'll take care of it.